core/src/main/scala/org/broadinstitute/dsde/rawls/methods/MethodConfigurationService.scala

```diff
@@ -475,7 +475,7 @@ class MethodConfigurationService(
                                                                               methodConfiguration,
                                                                               methodConfigResolver
       )
-      vmc <- entityProvider.expressionValidator.validateMCExpressions(methodConfiguration, gatherInputsResult)
+      vmc = entityProvider.expressionValidator.validateMCExpressions(methodConfiguration, gatherInputsResult)
```

Comment:
`validateMCExpressions` no longer returns a future, so this is a simple assignment now.